sections/semantics-embedded-content.include

```diff
@@ -11687,8 +10761,7 @@ red:89
 <h5 id="user-interface">User interface</h5>
 
   The <dfn element-attr for="mediaelements,video,audio"><code>controls</code></dfn> attribute is a
-  <a>boolean attribute</a>. If present, it indicates that the author has not provided a scripted
-  controller and would like the user agent to provide its own set of controls.
```

Comment:
Keep the part about "would like the user agent to provide its own set of controls."